packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -412,9 +412,16 @@ export class ProjectService extends BaseService {
         if (!isUserWithOrg(user)) {
             throw new ForbiddenError('User is not part of an organization');
         }
+        const { organizationUuid } = user;
         if (
-            user.ability.cannot('create', 'Job') ||
-            user.ability.cannot('create', 'Project')
+            user.ability.cannot(
+                'create',
+                subject('Job', { organizationUuid }),
```

Comment:
Job doesn't have any `subject` int he Ability spec, but it is good that we add it here,  in case we change the ability in the future. It doesn't cause any issues to provide additional subjects if they are not defined.